README.md

```diff
@@ -3,17 +3,20 @@
 ### This project is still under development.
 
 #### Using Comet Java SDK:
-* The latest released version is available in Maven Central: https://search.maven.org/search?q=ml.comet
-* Add the corresponding dependency to pom.xml.  For example:
+* The latest released version available in Maven Central:
+  [![Maven Central](https://maven-badges.herokuapp.com/maven-central/ml.comet/comet-java-sdk/badge.svg)](https://maven-badges.herokuapp.com/maven-central/ml.comet/comet-java-sdk) 
+   [![Build Status](https://travis-ci.com/comet-ml/comet-java-sdk.svg?branch=master)](https://travis-ci.com/github/comet-ml/comet-java-sdk)
+* Add dependency to pom.xml:
 ```
     <dependencies>
         <dependency>
             <groupId>ml.comet</groupId>
             <artifactId>comet-java-sdk</artifactId>
-            <version>1.0.7</version>
+            <version>1.0.12</version>
             <scope>system</scope>
             <systemPath>${project.basedir}/lib/comet-java-client-beta.jar</systemPath>
         </dependency>
     </dependencies>
 ```
-* create experiment - [example](/comet-examples/src/main/java/ml/comet/examples)
+* create experiment [example](comet-examples/src/main/java/ml/comet/examples/OnlineExperimentExample.java)
```

Comment:
in addition please add a short code snippet of how to use the SDK (init and send a parameter and metric)